app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -65,6 +65,11 @@ public function store(ReportbackRequest $request)
      */
     public function updateReportbackItems(Request $request)
     {
+        $this->validate($request, [
+            '*.rogue_reportback_item_id' => 'required',
```

Comment:
documentation is [here](https://laravel.com/docs/5.2/validation#validating-arrays)!